app/src/main/java/org/oppia/android/app/home/HomeActivity.kt

```diff
@@ -82,12 +89,30 @@ class HomeActivity :
     )
   }
 
-  override fun routeToRecentlyPlayed() {
-    startActivity(
-      RecentlyPlayedActivity.createRecentlyPlayedActivityIntent(
-        this,
-        internalProfileId
-      )
+  override fun routeToRecentlyPlayed(title: String) {
```

Comment:
Passing in title seems really error-prone--what if the language changes for a given screen and it gets recreated? Could we pass in an enum, instead? I was actually expecting that we'd be passing around RecentlyPlayedActivityTitle as that's one of the main benefits we get from introducing the enum.

Reply:
Updated